examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>I wrote another comment elsewhere about legacy data etc.. (I don't think that other comment is blocking tbh). That being said, at the very least we might need to offer some kind of insight on what is inside that CSV. Could we maybe cat it in a previous cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>agreed - or there should be some documentation around what cinnabar assumes is in an output csv. is the intention that this should be compatible with non-openfe outputs?</p>